src/du/du.rs

```diff
@@ -289,18 +278,25 @@ pub fn uumain(args: Vec<String>) -> i32 {
     };
 
     let convert_size = |size: u64| -> String {
+        let multiplier: u64 = if matches.opt_present("si") {
+            1000
+        } else {
+            1024
+        };
+        let units = vec![("T", 4), ("G", 3), ("M", 2), ("K", 1)];
```

Comment:
Do you mind adding support for `E` and `P` as well?  `Y` and `Z` should also be marked as `TODO` or something as they are larger than a `u64` can handle.

Reply:
Can do. 

Going to be offline a lot until easter so these changes are going to be slower.

Reply:
That's totally fine, take your time. :)